src/features/mutation_minigame/nodes/dna/Nucleotide.gd

```diff
@@ -0,0 +1,65 @@
+extends PanelContainer
+
+@export_enum("A", "G", "T", "C", "BLANK") var base: String = "BLANK"
+signal mutation(node, mutation)
+@export var bg_visible: bool = true:
+	set = set_bg_visible
+		
+func set_bg_visible(value):
+	if value:
+		self.remove_theme_stylebox_override("Panel")
+	else:
+		self.add_theme_stylebox_override("Panel",StyleBoxEmpty.new())
+
+var nitro_bases :Dictionary = {
+	"A": {
+		"Tooltip": "Adenine",
+		"Color": Color.GREEN,
+		"Text": "A"
+	},
+	"G": {
+		"Tooltip": "Adenine",
```

Comment:
I believe these tooltips are not accurate to other names. Also, can we make sure to have proper language support for these?

Reply:
Will do, think this should probably go into the global constants script as well.